spec/opera/operation/base_spec.rb

```diff
@@ -405,31 +405,45 @@ def step_1
       subject { operation_class.call(params: params, dependencies: dependencies) }
 
       context 'for validations' do
-        context 'when validation method returns unexpected object' do
+        context 'when validation step raises an exception' do
           let(:operation_class) do
             Class.new(Operation::Base) do
               validate do
                 step :validation_1
-                step :validation_2
               end
               step :step_1
 
               def validation_1
                 raise 'exception message'
               end
 
-              def validation_2
+              def step_1; end
+            end
+          end
+
+          it 'does not handle exception' do
```

Comment:
Maybe consistent
```suggestion
          it 'raises exception' do
```

as you used in L445?